code/pilotfile/plr.cpp

```diff
@@ -139,6 +139,8 @@ void pilotfile::plr_read_hud()
 
 	HUD_config.rp_flags = cfread_int(cfp);
 	HUD_config.rp_dist = cfread_int(cfp);
+	if (HUD_config.rp_dist < 0 || HUD_config.rp_dist >= RR_MAX_RANGES)
+		Error(LOCATION, "Player file has invalid radar range %d\n", HUD_config.rp_dist);
```

Comment:
Honestly, I believe they should be. If we can't get intelligible data out of the savefile, then we can't trust that the stuff we were able to read is accurate.